client/src/components/Reviews/ReviewSummary/index.jsx

```diff
@@ -3,22 +3,13 @@ import PropTypes from 'prop-types';
 import AverageRating from './AverageRating.jsx';
 import RatingBreakdown from './RatingBreakdown.jsx';
 import ProductBreakdown from './ProductBreakdown.jsx';
+import sharedFns from '../../SharedComponents/sharedFns.js';
 
 // TODO: add tests for AverageRating,
 // RatingBreakdown, and ProductBreakdown components
 
 const ReviewSummary = ({ reviewsMetadata, updateFilters }) => {
-  const ratings = reviewsMetadata.ratings || {};
-  const totalRatings = Object.values(ratings)
-    .reduce((prev, curr) => { return prev + Number(curr); }, 0);
-  const averageRating = totalRatings === 0 ? 0 : (Object.keys(ratings)
-    .reduce((prev, key) => {
-      let rating = Number(key);
-      let ratingCount = Number(ratings[key]);
-      return rating * ratingCount + prev;
-    }, 0) / totalRatings);
-  const recommendedCount = reviewsMetadata.recommended.true || 0;
-  const percentageRecommended = Math.floor(recommendedCount / totalRatings * 100);
+  let { totalRatings, percentageRecommended, averageRating } = sharedFns.calculateAverageRating(reviewsMetadata);
```

Comment:
Shouldn't the function here be called processReviewMetadata instead of calculateAverageRating? 
(That's the name for the function in the utilityFns.js)

Reply:
You are right! That's my bad. I think I might have updated the filenames/directory location and then I forgot to push my changes up. Thanks for catching that = )